torch/_refs/__init__.py

```diff
@@ -0,0 +1,588 @@
+import torch
+from torch._C import _add_docstr  # type: ignore[attr-defined]
+
+import torch._prims as prims
+import torch._prims.utils as utils
+from torch._prims import TensorLike as TensorLike
+
+from functools import reduce
+from enum import Enum
+from numbers import Number, Complex
+from typing import Sequence, Optional, Union, Callable
+
+# Experimental module containing prototype Python references for existing
+#   PyTorch operations.
+
+all = [
+    #
+    # Elementwise Binary References
+    #
+    "add",
```

Comment:
where's abs, that's the hard one 👺

Reply:
Just focused on elementwise binary for now